aws/src/main/java/org/apache/iceberg/aws/glue/GlueTableOperations.java

```diff
@@ -194,6 +195,21 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
     }
   }
 
+  /**
+   * Validate the Glue table is Iceberg table by checking its parameters
```

Comment:
can we update the javadoc here describing why `NoSuchIcebergTableException` is used? Also should add `@throws` line in the end

Reply:
Updated